app/components/invitation/invitation.js

```diff
@@ -22,7 +22,6 @@ var personUtils = require('../../core/personutils');
 var Invitation = React.createClass({
   propTypes: {
     invitation: React.PropTypes.object,
-    patientsComponent: React.PropTypes.component,
```

Comment:
So I guess it looks like we're just not actually using this in the Invitation component. Can we remove where this prop is passed in too?